lib/dogapi/v1/monitor.rb

```diff
@@ -231,6 +231,33 @@ def get_all_downtimes(options = {})
         end
       end
 
+      #
+      # HOST MUTING
```

Comment:
I think that would make sense if we interact with Hosts as objects - it appears that these methods are added to interact with the `/monitor` api interface.